dags/hivemind_telegram_etl.py

```
@@ -92,12 +106,18 @@ def processor(chat_info: tuple[str, str], community_id: str) -> None:
         if latest_date:
             # this is to catch any edits for messages of 30 days ago
             from_date = latest_date - timedelta(days=30)
+            logging.info(f"Started extracting from date: {from_date}!")
             messages = extractor.extract(from_date=from_date)
         else:
+            logging.info("Started extracting data from scratch!")
             messages = extractor.extract()
+
+        logging.info(f"Extracted {len(messages)} messages!")
         documents = transformer.transform(messages=messages)
+        logging.info(f"Messages transformed!")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)